### PR TITLE
Dataset description Funding converted to List

### LINKED ISSUE #314: validator fails when funding is a list rather than a single string
with the following in dataset_description.json:

   "Funding": [
        "TODO",
        "GRANT #1",
        "GRANT #2"
    ],

I get:

	1: JSON file is not formatted according the the schema. (code: 55 - JSON_SCHEMA_VALIDATION_ERROR)
		./dataset_description.json
			Evidence: .Funding should be string

when it's changed to a string rather than a list, the validation is fine.

validation should be changed to accept either a string or a list of strings.

### PR DESCRIPTION
**Pull request by suyashdb**
closes #314

### CODE REVIEW

File: validators/bids.js [hunk 1]
```diff
@@ -173,18 +173,19 @@ BIDS = {
 
         for (var f in fileList) {
             var completename = fileList[f].relativePath;
+            if(!completename.startsWith('/derivatives')) {
```
Comment (chrisgorgo) [inline validators/bids.js]: we should probably also exclude /code and /sourcedata

File: validators/schemas/dataset_description.json [hunk 2]
```diff
@@ -12,7 +12,10 @@
             "type": "string"
         },
         "Funding": {
-            "type": "string"
+            "items": {
+                "type": "string"
+            },
+            "type": "array"
```
Comment (chrisgorgo) [inline validators/schemas/dataset_description.json]: I wonder if for backward compatibility we should not allow both string and array of strings

**Comment by chrisgorgo** [comment]
I think you accidentally added `.idea` folder once again. Might be worth adding it to .gitignore.

File: validators/schemas/dataset_description.json (hunk 2 above)
Comment (suyashdb) [reply validators/schemas/dataset_description.json]: May be i didn't understand the above concern. Do you mean to say for backward  compatibility we **_should allow_** both string and array of strings. Above changes in this PR compels one to have _List_ _only_ ..

File: validators/bids.js (hunk 1 above)
Comment (suyashdb) [reply validators/bids.js]: @chrisfilo  True.. will add those two directory too.

File: validators/schemas/dataset_description.json (hunk 2 above)
Comment (chrisgorgo) [reply validators/schemas/dataset_description.json]: Apologies for misspeaking - "we should".

Comment (suyashdb) [reply validators/schemas/dataset_description.json]: Np. Will incorporate that!

**Comment by suyashdb** [comment]
@chrisfilo I have added **Funding** and **HowToAcknowldge** to be either list or list of string. 
I have seen **HowToAcknowldge** also in both formats. 
Let me know if you prefer **HowToAcknowldge** to be string only.

File: validators/schemas/dataset_description.json [hunk 3]
```diff
@@ -12,10 +12,13 @@
...
         },
         "Funding": {
-            "type": "string"
+            "items": {
+                "type": "string"
+            },
+            "type": ["array", "string"]
         },
         "HowToAcknowledge": {
-            "type": "string"
```
Comment (chrisgorgo) [inline validators/schemas/dataset_description.json:18]: doesn't this one also require specification that items need to be strings?

**Comment by chrisgorgo** [comment]
Could you merge master or rebase?

**Comment by suyashdb** [comment]
Ready for review.

File: validators/schemas/dataset_description.json (hunk 3 above)
Comment (chrisgorgo) [reply validators/schemas/dataset_description.json:18]: Any thoughts on my comment?

File: validators/schemas/dataset_description.json [hunk 4]
```diff
@@ -12,10 +12,13 @@
...
+            "type": ["string", "array"]
```
Comment (suyashdb) [inline validators/schemas/dataset_description.json]: yes, for "HowToAcknowledge" as well i think giving option of string and array of strings would make it backward compatible.

Comment (chrisgorgo) [reply validators/schemas/dataset_description.json]: I'm talking about adding
```
"items": {
                "type": "string"
},
```
to HowToAcknowledge the same way you did for Funding.

Comment (suyashdb) [reply validators/schemas/dataset_description.json]: Oh i see. I have pushed commit with those changes.

**Pull request merged**